app/jobs/fs/sync_box_job.rb

```diff
@@ -1,11 +1,27 @@
 module Fs
   class SyncBoxJob < ApplicationJob
-    def perform(box, from: Date.today - 1.week, to: Date.tomorrow)
+    def perform(box, from: Date.yesterday, to: Date.tomorrow, fs_client: FsEnvironment.fs_client, batch_size: 25)
```

Comment:
ja by som skúsil ten perform_limit napr. 2 (a prípadne upravíme na 1, keď uvidíme, ako to ide). Ideálne aj zmeniť frekvenciu na menej ako raz za 2h, ale najprv by som skúsil ten perform limit. Máme cca 7,5K FS boxov, takže to nie je zanedbateľné mnozstvo.